students/hojun/users/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
-from users.views import UserView
+from users.views import UserView, LogInView
 
 urlpatterns = [
-    path('', UserView.as_view())
+    path('user', UserView.as_view()),
+    path('login', LogInView.as_view())
```

Comment:
```suggestion
    path('signup', UserView.as_view()),
    path('login', LogInView.as_view())
```
회원가입의 최종적인 엔드포인트는 users/user 형태가 되겠네요. 엔드포인트만 봐도 어떤 기능을 하는 api인지 명확히 알 수 있어야합니다. 회원가입은 일반적으로 위와 같은 형태를 사용합니다.